src/Catel.Core/Catel.Core.Shared/Data/ModelBase.cs

```diff
@@ -312,6 +313,19 @@ protected internal bool LeanAndMeanModel
         public static bool DefaultDisableEventSubscriptionsOfChildValuesValue { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether usage of equality comparer for equals should be disabled.
+        /// <para />
+        /// The default value is <c>false</c>.
+        /// </summary>
+        /// <value><c>true</c> if usage of equality comparer for equals should be disabled; otherwise, <c>false</c>.</value>
+        protected bool DisableUsageOfEqualityComparerForEquals { get; set; }
```

Comment:
If someone does not want to use an equality comparer at all this prevents the call to GetType.